ircbot/plugins/translate.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+
+from langdetect import detect, detect_langs
+from ircbot import bot
+
+
+def translate(bot, text):
+    api_key = bot.config['Yandex']['translate_key']
+    lang = detect(text)
+    translate_url = "https://translate.yandex.net/api/v1.5/tr.json/translate?key={}&text={}&lang={}-{}".format(api_key, text, lang, bot.config["System"]["lang"])
```

Comment:
Does this line pass the linter? Looks rather long to me

Reply:
yeah whole thing got passed